File: sprixel/misc/value__S_number.js

// 
Act.types.value__S_number = function(){
    //throw keys(this.node);
    do { switch(this.phase) {
    case 0:
        this.phase = 1;
        return this.next = new Act(this.node.number, this);
    default: break;
    } } while (false);
    this.result = this.next.result;
    this.next = null;
    return this.invoker;
};
1;

File: vill/src/value__S_number.cpp
// value__S_number.cpp
#include "villCompiler.h"

Value *
value__S_number( struct villCompiler * vill, struct graph_node * node ) {
  Value * result = NULL;
  vill -> debug_flags & DEBUG_USER && fprintf( stderr, "<value__S_number>" );
  graph_node * number = ast_mapentry( node, "number" );
  assert( number != NULL );
  function_pointer Codegen;
  Codegen = (function_pointer) number -> data;
  assert( Codegen != NULL );
  result = Codegen( vill, number );
  vill -> debug_flags & DEBUG_USER && fprintf( stderr, "</value__S_number>" );
  return result;
}
